Documentation/policy/language.rst

```diff
@@ -34,6 +34,11 @@ can talk to each other. Layer 3 policies can be specified using the following me
   IP addresses or subnets into the policies. This construct should be used as a
   last resort as it requires stable IP or subnet assignments.
 
+* `DNS based`: Selects remote, non-cluster, peers using DNS names converted to
+  IPs via DNS lookups. It shares all limitations of the `CIDR based` rules
+  above. The current implementation simply polls the listed DNS targets without
+  regard for TTLs, and uses the response as-is.
```

Comment:
Perhaps it would be more explicit to rephrase the end of this sentence to something like the below?

`... without regard for TTLs, and allows traffics from IPs listed in the DNS responses.`

Reply:
sure. done